samples/instance-with-autoscaling-config.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright 2024 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+async function createInstanceWithAutoscalingConfig(instanceId, projectId) {
+  // [START spanner_create_instance_with_autoscaling_config]
+  // Imports the Google Cloud client library
+  const {Spanner, protos} = require('@google-cloud/spanner');
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const projectId = 'my-project-id';
+  // const instanceId = 'my-instance';
+
+  // Creates a client
+  const spanner = new Spanner({
+    projectId: projectId,
+  });
+
+  // Get the instance admin client
+  const instanceAdminClient = spanner.getInstanceAdminClient();
+
+  const autoscalingConfig =
+    protos.google.spanner.admin.instance.v1.AutoscalingConfig.create({
+      // Only one of minNodes/maxNodes or minProcessingUnits/maxProcessingUnits
```

Comment:
@harshachinta Please check if this comment is inline with other languages.

Reply:
Lets just have it as `Only one of minNodes/maxNodes or minProcessingUnits/maxProcessingUnits can be set` to give better clarity to customers. We have not added this in Java/Go though we have this in our spec doc.

Reply:
Updated